docs/DEPLOYMENT_SETUP.md

```diff
@@ -101,17 +101,20 @@ Store of AWS Service Manager. These groups have names formatted as `<environment
 
 The environment for each group is user-defined, and should be specified in the Elastic Beanstalk configuration
 as the environment variable `SSM_PREFIX`.
+The specific environment is user-defined, and should be specified in the Elastic Beanstalk configuration
+as the environment variable `SSM_ACTUAL_ENV_NAME`. This variable is optional.
 
 The list of categories can be inferred by looking at the file `.ebextensions/env-file-creation.config`.
 
 ### Core Features
 
-#### AWS Service Manager
+#### AWS Systems Manager
 
 | Variable     | Description                                                                                                           |
 | :----------- | --------------------------------------------------------------------------------------------------------------------- |
 | `SSM_PREFIX` | String prefix (typically the environment name) for AWS SSM parameter names to create a .env file for FormSG.          |
-| `SECRET_ENV` | String (typically the environment name) to be used in building of AWS Secrets Manager keys in different environments. |
+| `SECRET_ENV` | String (typically the environment name) to be used in building of AWS Secrets Manager keys in different environments. (`staging`, `prod`)|
+| `SSM_ACTUAL_ENV_NAME` | String (the specific environment name) to be used in building of AWS Secrets Manager keys in different environments. Optional. (`staging-alt`, `staging-alt2`)|
```

Comment:
non-blocking. Should this be `SSM_ENV_SITE_NAME` after the recent name change?

```suggestion
| `SSM_ENV_SITE_NAME` | String (the specific environment name) to be used in building of AWS Secrets Manager keys in different environments. Optional. (`staging-alt`, `staging-alt2`)|
```

Reply:
thanks for spotting! [9c0da8b](https://github.com/opengovsg/FormSG/pull/6181/commits/9c0da8b4af815242a53a4e3707cd6b23c644f86a)